### ISSUE TITLE
Define the directory structure

### ISSUE THREAD

**Issue by kof** [kof/feature-driven-architecture#4]
Or maybe it should be flexible?

**Comment by pgrekovich** [comment]
I think strict directory structure is better, because its help to describe some architecture concepts. And maybe in future, I think, can be added CLI for faster create features, shared components etc.

**Comment by kof** [comment]
Yeah, probably better, if someone needs to change it, they still can do, but we should have a clear guidance.
